src/pkg/selectors/groups.go

```diff
@@ -404,8 +506,16 @@ const (
 	GroupsChannelMessage groupsCategory = "GroupsChannelMessage"
 	GroupsLibraryFolder  groupsCategory = "GroupsLibraryFolder"
 	GroupsLibraryItem    groupsCategory = "GroupsLibraryItem"
+	GroupsList           groupsCategory = "GroupsList"
+	GroupsListItem       groupsCategory = "GroupsListItem"
+	GroupsPageFolder     groupsCategory = "GroupsPageFolder"
+	GroupsPage           groupsCategory = "GroupsPage"
 
 	// details.itemInfo comparables
+	GroupsInfoCreatedAfter   groupsCategory = "GroupsInfoCreatedAfter"
+	GroupsInfoCreatedBefore  groupsCategory = "GroupsInfoCreatedBefore"
+	GroupsInfoModifiedAfter  groupsCategory = "GroupsInfoModifiedAfter"
+	GroupsInfoModifiedBefore groupsCategory = "GroupsInfoModifiedBefore"
```

Comment:
These could use namespacing in order to not overlap with channel message consts.
```suggestion
	GroupsInfoLibraryItemCreatedAfter   groupsCategory = "GroupsInfoLibraryItemCreatedAfter"
	GroupsInfoLibraryItemCreatedBefore  groupsCategory = "GroupsInfoLibraryItemCreatedBefore"
	GroupsInfoLibraryItemModifiedAfter  groupsCategory = "GroupsInfoLibraryItemModifiedAfter"
	GroupsInfoLibraryItemModifiedBefore groupsCategory = "GroupsInfoLibraryItemModifiedBefore"
```